js/test/units/testunitfmt_usages.js

```diff
@@ -2732,6 +2750,7 @@ module.exports.testunitfmt_usages = {
 
         var uf = new UnitFmt({
             usage: "storageVolume",
+            measurementSystem: "imperial",
             length: "short",
```

Comment:
There should be some new tests that test that the usages for en-GB system is metric now except for certain usages. For example, drink volumes are imperial and person weight is imperial, but volume of a gas tank is liters and weight of a car is metric tonnes.

Reply:
I added more test cases with usage. Please re-review them